src/modules/commander/Commander.cpp

```diff
@@ -2254,6 +2279,8 @@ Commander::run()
 		/* Get current timestamp */
 		const hrt_abstime now = hrt_absolute_time();
 
+		checkEkfResetCounters();
```

Comment:
To be lazy you can combine this with the update check.

```suggestion
		if (_attitude_sub.update()) {
			checkEkfResetCounters();
		}
```

Unless you see further use of `vehicle_attitude` in commander coming it's probably also worth not keeping a copy.

Reply:
Reminder for later, add an attitude validity flag.